account/views.py

```
@@ -60,11 +80,76 @@ def post(self, request):
+        
+        refresh_token = request.data.get("refresh")
+        #### 1
+        if not refresh_token:
+            return Response(
+                {"detail": "no refresh token."}, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        try:
+        #### 2
+            RefreshToken(refresh_token).verify()
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)